exonum-java-binding-core/src/main/java/com/exonum/binding/storage/indices/ConfigurableRustIter.java

```diff
@@ -32,8 +32,10 @@
 final class ConfigurableRustIter<E> extends AbstractNativeProxy implements RustIter<E> {
 
   private final LongFunction<E> nextFunction;
+  // todo: This view is kept only for diagnostic purposes (to put in error message) —
```

Comment:
Views don't have `toString` overridden, so would it be helpful to log things like `com.exonum.binding.storage.database.Fork@131ef10`? Or maybe it would make sense to return `View#getViewNativeHandle` for diagnostic purposes? If no to both questions, then I'd remove this field.